### DISCUSSION TITLE
weekly/2024-07-28

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#278] [Announcements]
# weekly/2024-07-28

① Stack Overflow 2024 开发者调查报告 ② React 2023 调查结果 ③ Sendune - HTML 邮件制作工具 ④ hash-wasm - 哈希函数库 ⑤ ky - 基于 fetch 的 HTTP 客户端 ⑥ 英：前端构建概述 ⑦ Album AI - AI 图片检索 ⑧ AI Emoji Search

https://sugarat.top/weekly/2024-07-28.html

**Comment by XiaofengdiZhu** [comment]
🥄🍚🍬
